### DISCUSSION TITLE
Playability

### DISCUSSION THREAD

**Discussion by Fequois** [DiplomacyTeam/Bannerlord.Diplomacy#3] [General]
Hey @zijistark ! Just wondering if this mod builds, if it works at all and for which game version.

**Comment by iwanadiefast** [comment]
it does but it's a little weird, sometimes it doesn't want to load a save (still have no idea why. i'd need to debug irt to see what's going on i guess). 
just select the branch and the version you want to build (1.5.7 or .8_release) and voila. i'm currently using the dev branch. maybe i'll switch to the main later, dunno. i don't really know what i'm doing sometimes but yea
confia
